Softeq.XToolkit.Permissions/PluginPermissionStatusExtensions.cs

```diff
@@ -14,6 +14,7 @@ public static PermissionStatus ToPermissionStatus(this PluginPermissionStatus pe
             {
                 PluginPermissionStatus.Denied => PermissionStatus.Denied,
                 PluginPermissionStatus.Disabled => PermissionStatus.Denied,
+                PluginPermissionStatus.Limited => PermissionStatus.Granted,
```

Comment:
Maybe `Restricted` would fit better?
In which cases can we get `Limited` for each permission except `Photos`?